lib/fog/parsers/terremark/vapp.rb

```diff
@@ -11,46 +11,46 @@ def reset
         def start_element(name, attributes)
           super
           case name
-            when 'Link'
-              link = extract_attributes(attributes)
-              @response['Links'] << link
-            when 'OperatingSystemSection'
-              @in_operating_system = true
-            when 'VApp'
-              vapp = extract_attributes(attributes)
-              @response.merge!(vapp.reject {|key,value| !['href', 'name', 'size', 'status', 'type'].include?(key)})
+          when "Link"
+            link = extract_attributes(attributes)
+            @response["Links"] << link
+          when "OperatingSystemSection"
+            @in_operating_system = true
+          when "VApp"
+            vapp = extract_attributes(attributes)
+            @response.merge!(vapp.reject {|key,value| !["href", "name", "size", "status", "type"].include?(key)})
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.